### PR TITLE
build: Disable pnpm update checks

### PR DESCRIPTION
**Pull request by tylerbutler**
Disables pnpm's update check/notification using the [update-notifier](https://pnpm.io/npmrc#update-notifier) setting in `.npmrc`. Since we use corepack to ensure we use the same version of pnpm everywhere, we don't want individuals on the team to be prompted to update their version of pnpm.

I also noticed that there were some projects using pnpm without an npmrc, so I added one to those projects.

### CODE REVIEW

File: common/lib/common-definitions/.npmrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+strict-peer-dependencies=true
```
Comment (alexvy86) [inline common/lib/common-definitions/.npmrc:1]: Missed `engine-strict` here?

**Review by alexvy86 — APPROVED** [review]
One small thing. 🚀

File: common/lib/common-definitions/.npmrc (hunk 1 above)
Comment (tylerbutler) [inline common/lib/common-definitions/.npmrc:1]:
```suggestion
engine-strict=true
strict-peer-dependencies=true
```

Comment (tylerbutler) [reply common/lib/common-definitions/.npmrc:1]: Good catch!

**Pull request merged**